### PR TITLE
Change instances of hardcoded categories

### PR DESCRIPTION
**Pull request by collinkan**
Categories are now retrieved from Firestore.

### CODE REVIEW

File: my/src/components/Dialog.vue [hunk 1]
```diff
@@ -16,7 +16,12 @@
 import Vue from 'vue';
 
 export default Vue.extend({
-  props: ['open', 'title', 'body', 'value'],
+  props: {
+    title: String, 
+    body: String, 
+    value: Boolean, 
+    persist: {type: Boolean, default: false}
```
Comment (amirafzali) [inline my/src/components/Dialog.vue:23]: great variable naming 🔥

File: my/src/data.ts [hunk 2]
```diff
@@ -286,29 +286,23 @@ const workshops = [
   'Computer Vision with OpenCV',
 ];
 
-const challenges = [
```
Comment (amirafzali) [inline my/src/data.ts:289]: 👋

**Review by amirafzali — APPROVED** [review]
Everything looks good, thanks Collin!

**Review by enricoflorentino — APPROVED** [review]
clean, the dialog + submit components work as expected. nice!

**Pull request merged**